msteams-platform/messaging-extensions/what-are-messaging-extensions.md

```diff
@@ -1,74 +1,75 @@
 ![messaging extension action command task module](~/assets/images/task-module.png)
 
 ### Search commands
 
-Search commands allow your users to search an external system for information (either manually through a search box, or by pasting a link to a monitored domain into the compose message area), then insert the results of the search into a message. In the most basic search command flow, the initial invoke message will include the search string the user submitted. You'll respond with a list of cards and card previews. The Teams client will render the card previews in a list for the end user to select from. When the user selects a card, the full-size card will be inserted into the compose message area.
-
-They can be triggered from the compose message area or the command box. Unlike action commands, they cannot be triggered from a message.
+Search commands allow the users to search an external system for information either manually through a search box, or by pasting a link to a monitored domain into the compose message area, and insert the results of the search into a message. In the most basic search command flow, the initial invoke message includes the search string that the user submitted. You respond with a list of cards and card previews. The Teams client renders a list of card previews for the user. When the user selects a card from the list, the full-size card is inserted into the compose message area.
 
+The cards are triggered from the compose message area or the command box and not triggered from a message. They can not be triggered from a message.
+The following image displays the messaging extension search command task module:
 ![messaging extension search command](~/assets/images/search-extension.png)
 
-### Link unfurling
+> [!NOTE]
+> For more information on cards, see [what are cards](../task-modules-and-cards/what-are-cards.md)
 
-You also have the option to invoke your service when a URL is pasted in the compose message area. This functionality, known as **link unfurling**, allows you to subscribe to receive an invoke when URLs containing a particular domain are pasted into the compose message area. Your web service can "unfurl" the URL into a detailed card, providing more information than the standard website preview card. You can even add buttons to allow your users to immediately take action without leaving the Microsoft Teams client.
+## Link unfurling
 
-## Get Started
+A web service is invoked when a URL is pasted in the compose message area. This functionality is known as **link unfurling**. You can subscribe to receive an invoke when URLs containing a particular domain are pasted into the compose message area. Your web service can "unfurl" the URL into a detailed card, providing more information than the standard website preview card. You can add buttons to allow the users to immediately take action without leaving the Microsoft Teams client. 
 
```

Comment:
Could you please also add screenshot for link unfurl? I found pretty neat gif, check if you can use that. 

https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/msgext-link-unfurling-reddit/csharp/doc/images/Preview.gif